frontend/public/components/overview.jsx

```diff
@@ -139,14 +140,15 @@ class OverviewDetails extends React.Component {
 
   filterItems(items) {
     const {filterValue} = this.state;
+    const {currentSelection} = this.props;
 
     if (!filterValue) {
       return items;
     }
 
     const filterString = filterValue.toLowerCase();
     return _.filter(items, item => {
-      return fuzzy(filterString, _.get(item, 'metadata.name', ''));
+      return fuzzy(filterString, _.get(item, 'metadata.name', '')) || _.get(item, 'metadata.uid') === _.get(currentSelection, 'metadata.uid');
```

Comment:
The only problem I have with that is that it will cause the filter input to jump across the page as you are typing.

Reply:
I'm fine leaving it as you have it, at least for now. We can always change it.

@beanh66 FYI, we're not show what to do if a filter hides the resource you have open. Filtering it out would cause the input to jump as you type since it would close the sidebar.

Reply:
I wonder if we should disable the filter bar while the side panel is open? I agree the jump may be weird otherwise and it seems like we may mislead some users into thinking they are filtering that side content too, when they are not. Thoughts @openshift/team-ux-review?